### DISCUSSION TITLE
GitHub Branch assignment

### DISCUSSION THREAD

**Discussion by vmwielgus** [ua-gist604b-f24/syllabus#7] [General]
Thought I was doing this assignment correctly, then received two error emails, one while creating my-branch and the second after the pull request. Could anyone verify what the end result should look like? Thanks
![github_fail_my_branch](https://github.com/user-attachments/assets/b48868a5-1b89-4990-a61e-bb5518298477)
![github_fail_master](https://github.com/user-attachments/assets/6dd4062f-d17f-41b2-8ef3-fa0431dbc4e6)

**Comment by aaryno** [comment]
I'm not sure of what the CI failure is but you can get more information by Viewing the Worklflow Run and expanding any errors you see.

As far as your assignment goes, It is almost done. You only need to Open a Pull Request for this https://github.com/ua-gist604b-f24/1-0-github-branch-vmwielgus/tree/my-branch.
